src/Docker/docker-compose.test-local.yml

```diff
@@ -30,7 +30,7 @@ services:
       - PUBLIC_FOLDER=/usr/src/app/dist
       - AUTH0_ISSUER_URL=https://dev-326tk4zu.us.auth0.com/
       - AUTH0_AUDIENCE=https://www.vectoratlas.org
-      - TOKEN_KEY=<token-key>
+      - TOKEN_KEY=LmUVODxYxi74zcpxk1rha+H7QUo=
```

Comment:
We shouldn't check secrets into the code base - I need to think about the strategy for loading this here and how the deployment process needs to be updated. We'll merge this for now and I'll think about the approach afterwards.